### ISSUE TITLE
🐛 [Bug] Map - users without descriptions have null on map shown

### ISSUE THREAD

**Issue by ulfgebhardt** [Ocelot-Social-Community/Ocelot-Social#5979]
## 🐛 Bugreport

Map - users without descriptions have null on map shown

![image](https://user-images.githubusercontent.com/1238238/219358622-8bd258c1-9213-457f-857b-09b632fddbc7.png)

**Comment by Tirokk** [comment]
I added this to https://github.com/Ocelot-Social-Community/Ocelot-Social/issues/5901 @ulfgebhardt

**Comment by Elweyn** [comment]
Hi @ulfgebhardt, @Tirokk, @Mogge.

I looked into this and I think I found the line where the description is written in the PopUp of the User:
webapp/pages/map.vue line 237 - 266 and in this lines the line 259 to 266
```
description +=
          e.features[0].properties.about && e.features[0].properties.about.length > 0
            ? `
            <hr>
            <div>
              ${e.features[0].properties.about}
            </div>`
            : ''
```

I made a bit more tests with the UI and got this:
- If ``about`` is the default value we get what we see at the top
- if ``about`` is filled we get the information of about
- if ``about`` has been emptied then we don't see null in the description!

Can it be that the default value of about is 'null'?
